src/FakeItEasy/Fake.cs

```diff
@@ -47,7 +48,7 @@ public static IEnumerable<ICompletedFakeObjectCall> GetCalls(object fakedObject)
         /// </summary>
         /// <param name="fakedObject">The faked object to clear the configuration of.</param>
         [SuppressMessage("Microsoft.Naming", "CA1720:IdentifiersShouldNotContainTypeNames", MessageId = "object", Justification = "The term fake object does not refer to the type System.Object.")]
-        [Obsolete("ClearConfiguration will be removed in version 8.0.0. Prefer to discard the fake and create a new one.")]
+        [Obsolete("ClearConfiguration will be removed in version 8.0.0. Use Fake.Reset, or discard the fake and create a new one.")]

```

Comment:
```suggestion
        [Obsolete("ClearConfiguration will be removed in version 9.0.0. Use Fake.Reset, or discard the fake and create a new one.")]
```

? I know we warned people about 8.0.0, but then we blew past it (my bad) and now the 8.1 people have been using `ClearConfiguration` all the livelong day and I'd feel bad if 8.2 bit them.